testing/trino-testing/src/main/resources/io/trino/testing/containers/hadoop-run.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if test $# -gt 0; then
+    echo "$0 does not accept arguments" >&2
+    exit 32
+fi
+
+set -x
+
+HADOOP_INIT_D=${HADOOP_INIT_D:-/etc/hadoop-init.d/}
+
+echo "Applying hadoop init.d scripts from ${HADOOP_INIT_D}"
+if test -d "${HADOOP_INIT_D}"; then
+    for init_script in "${HADOOP_INIT_D}"*; do
+        "${init_script}"
+    done
+fi
```

Comment:
do we need this. Are there any scripts there?

Reply:
Will remove it. It's not used in current code.